### ISSUE TITLE
(PT015) REP Board Meeting "bookmark"

### ISSUE THREAD

**Issue by altrdgenetics** [CurrentlyNotSet/SERB#206]
so someone doesnt know how the old system worked. When selecting a recommendation for the boardmeeting there is text in there like ' <<Polling_Period>> '. SERB was under the impression that this was a bookmark not working, when in actuality it was just text with tags around it to notify the user they need to enter a date. 

![image](https://cloud.githubusercontent.com/assets/8847344/23480843/69367102-fe97-11e6-84ce-d1885de420ab.png)

After emailing Justin back and forth, it may turn into a replace text on selecting the bookmark. We are unsure yet.

**Comment by altrdgenetics** [comment]
Still waiting on response from Justin

**Comment by altrdgenetics** [comment]
In the meeting, still do not have the answer on what they are wanting to do.

**Comment by altrdgenetics** [comment]
In the meeting brought it up that we have yet to receive the list. They are aware of it and expressed their disappointment as well.

**Comment by altrdgenetics** [comment]
JUSTIN EMAIL:

> Angela is hoping to get this list to me tomorrow. It is the next item on the list on her desk. She found another issue that took precedence over getting this to me.

**Comment by parkjohnston** [comment]
Soooo is this "next on the list" item really that important has it is yet another several days?

May want to start hitting back on tickets that have been open this long as not important....we can't just sit around waiting for them for 1+ month(s)

**Comment by altrdgenetics** [comment]
Email from Justin:

> Angela is in the middle of working on this today. She is going through an excel list that I provided her last fall of all the Board Meeting Recommendations. She is tweaking them, adding some new entries, and noting the bookmarks. This updated excel file will be coming soon.

**Comment by altrdgenetics** [comment]
So i got this coded out complete but have another question still... Apparently they cant even stay consistent in their own naming convention. Was told to replace <PetitionerIntervenor>>. So which party is it??? emailed Justin.

**Comment by altrdgenetics** [comment]
talked to justin on the phone... he has no idea either. he also found maybe some other bookmark. He will be getting back to us

**Comment by altrdgenetics** [comment]
he got us the info, so now this is done

**Comment by altrdgenetics** [comment]
4hrs of labor
